components/fxa-client/android/src/main/java/mozilla/appservices/fxaclient/MetricsParams.kt

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.appservices.fxaclient
+
+data class MetricsParams(
+    val flowId: String? = null,
+    val flowBeginTime: String? = null,
```

Comment:
This is a timestamp at first and then we turn it into a "string" for the url. Not sure if we want to change this for the consumers.